### PR TITLE
[tests-only][full-ci]Search files through content by sharee

### PR DESCRIPTION
**Pull request by nabim777**
## Description
This PR adds the API tests for full search api. The scenarios added in this PR are
- sharee search files using content

## Related Issue
- https://github.com/owncloud/ocis/issues/6606#event-9635806756

## Motivation and Context
- there was no test coverage for api test for sharee search files using content. so, this PR covers the require scenario test case

## How Has This Been Tested?
- test environment:
- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiFullTextSearch/search.feature [hunk 1]
```diff
@@ -25,3 +25,23 @@ Feature: full text search
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario Outline: sharee search files using content
```
Comment (grgprarup) [inline tests/acceptance/features/apiFullTextSearch/search.feature:30]:
```suggestion
  Scenario Outline: sharee searches files by content
```

File: tests/acceptance/features/apiFullTextSearch/search.feature [hunk 2]
```diff
@@ -25,3 +25,23 @@ Feature: full text search
...
+    Given using <dav-path-version> DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
+    And user "Alice" has created folder "uploadFolder"
+    And user "Alice" has uploaded file with content "hello world" to "uploadFolder/file1.txt"
+    And user "Alice" has uploaded file with content "Namaste nepal" to "uploadFolder/file2.txt"
+    And user "Alice" has uploaded file with content "hello nepal" to "uploadFolder/file3.txt"
```
Comment (grgprarup) [inline tests/acceptance/features/apiFullTextSearch/search.feature:36]: Refer to https://github.com/owncloud/ocis/pull/6709#discussion_r1266270454

**Comment by saw-jan** [comment]
@nabim777 update the PR title and the description. (should be `content`)

File: tests/acceptance/features/apiFullTextSearch/search.feature (hunk 2 above)
Comment (nabim777) [reply tests/acceptance/features/apiFullTextSearch/search.feature]: Ok

**Review by saw-jan — COMMENTED** [review]
LGTM

**Review by SagarGi — COMMENTED** [review]
LGTM :+1:

**Review by phil-davis — APPROVED** [review]
LGTM

**Pull request merged**